content/doc/developer/internationalization/i18n-jelly-views.adoc

```diff
@@ -36,4 +36,5 @@ Then you could add, for example, a Chinese localization file simply as:
 message=\u6D88\u606F
 ----
 
-In property files all non-ASCII characters need to be converted into hexcode. Modern IDEs do it automatically, so you can be just writing localizations in the target languages there.
+Properties files should be UTF-8 encoded, at least for Core or Plugins which already use Java 11 and a core version greater than 2.358. More details can be found in link:https://issues.jenkins.io/browse/JENKINS-41729[JENKINS-41729].
+For older plugins, all non-ASCII characters need to be converted into hexcode. Modern IDEs do it automatically, so you can be just writing localizations in the target languages there.
```

Comment:
```suggestion
Properties files should be UTF-8 encoded, at least for Core or Plugins which already use Java 11 and a core version greater than 2.358.
More details can be found in link:https://issues.jenkins.io/browse/JENKINS-41729[JENKINS-41729].
For older plugins, all non-ASCII characters need to be converted into hexcode.
Modern IDEs do it automatically, so you can be just writing localizations in the target languages there.
```
small formatting suggestion, not necessary for approval though.
thanks for this @StefanSpieker !